modules/mobile/app/controllers/mobile/v0/pre_need_burial_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'chip/service'
+require 'preneeds/service'
+
+module Mobile
+  module V0
+    class PreNeedBurialController < ApplicationController
+      FORM = '40-10007'
+
+      def create
+        validate!
+
+        @form = ::Preneeds::BurialForm.new(burial_form_params)
```

Comment:
Just as a very small point of style, I'd suggest not making these instance variables if you only need local scope. It's just easier to read the code when you know that the variables aren't being used elsewhere.